src/Halcyon/MemoryCacheManager.php

```diff
@@ -0,0 +1,19 @@
+<?php namespace October\Rain\Halcyon;
+
+use Config;
+use Illuminate\Cache\CacheManager;
+use Illuminate\Contracts\Cache\Store;
+
+class MemoryCacheManager extends CacheManager
+{
+    public function repository(Store $store)
+    {
+        $repository = new MemoryRepository($store);
+        return $repository;
+    }
+
+    public static function isEnabled()
+    {
+        return !Config::get('cache.disableRequestCache', false);
```

Comment:
Are you able to use the `$this->app->config` methodology for accessing the config value instead of pulling in the Config facade? It's used in other places in this repo so I'd prefer to use it here too

Reply:
I don't think I can since this is a static function, unless I'm missing something? I used the facade because I had seen that other places as well.

Reply:
Fair enough, you'd have to implement it as a facade which would be a bit ridiculous